.github/CODEOWNERS

```diff
@@ -1,10 +1,23 @@
+# Console
+/src/Symfony/Component/Console/Logger/ConsoleLogger.php @dunglas
+# DependencyInjection
+/src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php @dunglas
+# HttpKernel
+/src/Symfony/Component/HttpKernel/Log/Logger.php @dunglas
 # LDAP
 /src/Symfony/Component/Ldap/* @csarrazi
 # Lock
 /src/Symfony/Component/Lock/* @jderusse
 # Messenger
 /src/Symfony/Bridge/Doctrine/Messenger/* @sroze
 /src/Symfony/Component/Messenger/* @sroze
+# PropertyInfo
+/src/Symfony/Component/PropertyInfo/* @dunglas
+/src/Symfony/Bridge/Doctrine/PropertyInfo/* @dunglas
+# Serializer
```

Comment:
Should be LDAP, right?

Reply:
Faulty copy/paste ^^ (should be Serializer). Fixed.